portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +24,48 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+ Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
+
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+
+ArrayList<String> nums = new ArrayList<String>();
```

Comment:
```suggestion
List<String> nums = new ArrayList<String>();
```

Better to use an interface than concrete type.